### PR TITLE
[2.11.x] DDF-3279 Updates itests to use embedded solr

### PR DESCRIPTION
**Pull request by vinamartin**
#### What does this PR do?
Updates itests to use embedded solr + disables tests that require http solr client. 

#### Who is reviewing it? 
@clockard @tbatie @emanns95 @Lambeaux 

#### Choose 2 committers to review/merge the PR. 
@figliold 
@lessarderic

#### How should this be tested? (List steps with links to updated documentation)
Full build with all itests passing 

#### What are the relevant tickets?
[DDF-3279](https://codice.atlassian.net/browse/DDF-3279)

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSolrCommands.java [hunk 1]
```diff
@@ -73,12 +74,14 @@ public void tearDown() {
     cleanUpBackups(CATALOG_CORE_NAME);
   }
 
+  @Ignore
```
Comment (clockard) [inline distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSolrCommands.java]: Why don't these work anymore?

Comment (vinamartin) [reply distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSolrCommands.java]: I thought these were using the http solr client, if I'm wrong I can re-enable these.

**Comment by vinamartin** [comment]
build now

File: distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSolrCommands.java (hunk 1 above)
Comment (clockard) [reply distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSolrCommands.java]: So the solr commands don't work when using the embedded instance? If so we might want to make a not of that somewhere if we don't already.

File: distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java [hunk 2]
```diff
@@ -2599,6 +2599,7 @@ public void testIngestIgnore() throws Exception {
         .post(REST_PATH.getUrl());
   }
 
+  @Ignore
```
Comment (lessarderic) [inline distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java]: ❓ Can we add a ticket number everywhere we `@Ignore` a test?

File: distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSolrCommands.java (hunk 1 above)
Comment (lessarderic) [reply distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSolrCommands.java]: The backup command uses the HTTP endpoint, so it doesn't work when using the Embedded Solr. The documentation does mention that Embedded Solr must be backed up manually. I couldn't find any reference to the `solr:backup` command in the docs though. @ricklarsen?

**Comment by vinamartin** [comment]
I've added TODOs @lessarderic

**Comment by vinamartin** [comment]
@emanns95 I've updated the PR, do you mind testing this?

**Comment by vinamartin** [comment]
build now

**Pull request merged**